godtools/App/Features/FeaturedLessons/Domain/Entities/FeaturedLessonDomainModel.swift

```diff
@@ -14,6 +14,7 @@ struct FeaturedLessonDomainModel: LessonListItemDomainModelInterface {
     let bannerImageId: String
     let dataModelId: String
     let name: String
+    let nameLanguageDirection: LanguageModel.Direction
```

Comment:
Can you change this type to ```LanguageDirectionDomainModel```.

Reply:
Makes sense, thanks!  I do get mixed up when properties are so similar.  Just need to remember to think through the layers.

Reply:
Thanks, I'll need to remove the LanguageModel from LanguageDirectionDomainModel too since it really shouldn't know about the data layer.  Instead I'll create an extension on LanguageModel to convert to the LanguageDirectionDomainModel.